config.libsonnet

```diff
@@ -58,8 +58,8 @@
     // servers under some non-root path.
     grafanaPrefix: '',
 
-    //We'd like to allow users to opt-in to multiCluster dashboards (should default: false)
-    showMultiCluster: true,
+    //Opt-in to multiCluster dashboards by overriding this and the clusterLabel
```

Comment:
Should we group the clusterLabel the this flag together in the config?

Reply:
Yea, that's prob helpful